src/lib.rs

```diff
@@ -1157,7 +1156,7 @@ pub mod contract {
             let swap_amount = TokenAmount::new(amount);
             approve!(client, TokenRef, token_x, dex, amount, alice);
 
-            let target_sqrt_price = SqrtPrice::new(990157831158286630888963);
+            let target_sqrt_price = SqrtPrice::new(MAX_SQRT_PRICE);
```

Comment:
It's `slippage_test`, so I am not sure whether `MAX_SQRT_PRICE` is the proper target. You probably should use something closer to the resulting sqrt price